src/js/components/ComboBox/Combobox.tsx

```diff
@@ -26,6 +26,7 @@ import { Button } from '../Button';
 import { Checkbox } from '../Checkbox';
 import { Field } from '../Field';
 import { Icon } from '../Icon';
+import { isEqual } from 'lodash';
```

Comment:
Apparently we already have lodash as a dependency for TeamSnap UI, but I dont think depending on that is a good idea ultimately. We'll probably want to remove this at some point, but its likely way beyond the scope of this PR.

Reply:
Absolute agree with you. Might be a good thing to visit during cooldown. I think for now in order to keep pushing through, if okay, I'm going to opt to keep as is.